### ISSUE TITLE
Error: Unknown builtin plugin "symbolId" specified.

### ISSUE THREAD

**Issue by curly210102** [nyanyani/rollup-plugin-svg-sprite-loader#2]
<img width="889" alt="Screen Shot 2021-10-23 at 8 12 11 AM" src="https://user-images.githubusercontent.com/76864176/138534840-59ee58c8-b78c-4d2e-997c-bec5840214c9.png">

``` js
// rollup.config.json

plugins: [{
  svgSprite({
      symbolId: (filePath) => path.basename(filePath) + hash
  }),
}]
```

It seems lose the dependence of svgo plugin symbolId in package.json.

**Comment by nyanyani** [comment]
Sorry, i haven't implemented this option correctly yet xD. Acctually i've published a worng version of README which was borrowed from [svg-sprite-loader](https://github.com/JetBrains/svg-sprite-loader#symbol-id), and you were misleaded by my fault.

I will fix it ASAP, thanks for you issue :D.

**Comment by nyanyani** [comment]
> Sorry, i haven't implemented this option correctly yet xD. Acctually i've published a worng version of README which was borrowed from [svg-sprite-loader](https://github.com/JetBrains/svg-sprite-loader#symbol-id), and you were misleaded by my fault.
> 
> I will fix it ASAP, thanks for you issue :D.

A new version has been published, check this out  [Release v0.0.2](https://github.com/nyanyani/rollup-plugin-svg-sprite-loader/releases/tag/v0.0.2).

**Comment by equinusocio** [comment]
I think this issue can be closed, the error is gone
